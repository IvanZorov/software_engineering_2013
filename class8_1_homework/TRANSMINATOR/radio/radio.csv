W0WX,5.43,AM
9TSMD,21.23,SSB
ARW,11.5,AM
BBC-W,7.33,AM
DL5RT,27.12,FM
HG7TRX,18.33,AM
J3SMP,28.3,SSB
K6JAW,24.2,AM
LZ0BGN,439.200,FM
LZ0DOE,439.000,SSB
LZ0UHF,439.4,FM
LZ1KZX,14.07,SSB
LZ2KSC,14.27,SSB
LZ2KSC,144.33,AM
LZ3WE-R,438.75,FM
PY7WSD,18.04,AM
RL3RFT,24.3,SSB
S7VTD,28.6,FM
SDR-T,10.05,AM
SDWR-01,111.45,AM
SJ5F,29.1,SSB
SV1TYT,24.1,SSB
TA2AHT,144.45,AM
TA4SVX,145.600,FM
TS2KH,434.45,SSB
TY4GH,21.25,AM
W1AW,7.164,SSB
YO2YA,28.7,AM
YU1JY,145.225,SSB
YU3SD,144.35,SSB
Z7JOF,28.13,SSB
