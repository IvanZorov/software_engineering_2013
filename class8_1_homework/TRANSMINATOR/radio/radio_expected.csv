"DL5RT","27.12","FM"
"S7VTD","28.6","FM"
"TA2AHT","145.45","FM"
"TA4SVX","145.600","FM"